packages/core-components/src/lib/unsorted/viz/Chart.svelte

```diff
@@ -821,7 +821,7 @@
 	$: if (data?.error) error = data.error.message;
```

Comment:
Should this be changed to `$: if ($data?.error) error = data.error.message ?? data.error;` as well?